script/setup_host_fedora.sh

```diff
@@ -1,12 +1,11 @@
 #!/bin/bash
 set -eux -o pipefail
-DNF_OPTS="-y --setopt=install_weak_deps=False --setopt=tsflags=nodocs --exclude=kernel,kernel-core"
-RPMS="bats git-core glibc-static golang jq libseccomp-devel make"
+DNF=(dnf -y --setopt=install_weak_deps=False --setopt=tsflags=nodocs --exclude="kernel,kernel-core")
+RPMS=(bats git-core glibc-static golang jq libseccomp-devel make)
```

Comment:
Can't we just keep this as it was, and just use `"$RPM"` when using it? Won't that be simpler and solve the warning too?

Reply:
No, it doesn't work this way.

```console
[kir@kir-tp1 runc]$ RPMS="bash top"
[kir@kir-tp1 runc]$ sudo dnf update "$RPMS"
<....>
Repositories loaded.
Failed to resolve the transaction:
No match for argument: bash top
You can try to add to command line:
  --skip-unavailable to skip unavailable packages
```

Reply:
For more details, please see https://www.shellcheck.net/wiki/SC2086.

Reply:
The alternatives are:
 - keep the "shellcheck disable SC2086" annotation;
 - inline the list of rpms.

Reply:
Ohh, right! I'm usually on the `eval` team rather than arrays (I think that should solve it too). But yeah, between eval and this, I think both are quite similar. Let's keep it like this :)